src/game.cpp

```diff
@@ -1482,7 +1482,9 @@ bool game::do_turn()
     monmove();
     update_stair_monsters();
     u.process_turn();
-    if (u.moves < 0) {
+    int redraw = ( int )OPTIONS["REDRAW_FREQUENCY"];
```

Comment:
I realize this seems kind of redundant, but it's used to fix some undefined meaning of the "!=" operator errors that my compiler flagged as well as the basic warning that type float (which is apparently how the options value is retrieved) might not fit into type int.